From: Brendan Jackman <brendan.jackman@linux.dev>
Date: Sat, 25 Jul 2026 12:34:29 +0000
Subject: [PATCH] 
 mm-page_alloc-fix-non-movable-reclaim-storm-in-defrag_mode-fix

fix try_to_compact_pages() kerneldoc

Link: https://lore.kernel.org/DK7NM9RPUJOD.11PNJJ5N2OBED@linux.dev
Signed-off-by: "Brendan Jackman" <brendan.jackman@linux.dev>
Cc: Johannes Weiner <hannes@cmpxchg.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 mm/compaction.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/compaction.c b/mm/compaction.c
index 0568623d9384d..a049415512c67 100644
--- a/mm/compaction.c
+++ b/mm/compaction.c
@@ -2823,7 +2823,7 @@ static enum compact_result compact_zone_order(struct zone *zone, int order,
 /**
  * try_to_compact_pages - Direct compact to satisfy a high-order allocation
  * @gfp_mask: The GFP mask of the current allocation
- * @order: The order of the current allocation
+ * @order: The order to try to make available
  * @alloc_flags: The allocation flags of the current allocation
  * @ac: The context of current allocation
  * @prio: Determines how hard direct compaction should try to succeed
-- 
2.47.3

